grails-app/services/com/unifina/service/EthereumService.groovy

```diff
@@ -0,0 +1,29 @@
+package com.unifina.service
+
+import com.unifina.signalpath.blockchain.Web3jHelper
+import org.web3j.abi.datatypes.Utf8String
+import org.web3j.protocol.Web3j
+
+class EthereumService {
```

Comment:
Feels like these should be in `CommunityService`, as all these methods are related to Community smart contracts and not Ethereum in general.

Reply:
This makes testing easier. Perhaps a better name, something like CommunityEthereumService

Reply:
CommunityService, ProductCommunityService, CommunityProductService, I think.